src/lib/schemas.ts

```diff
@@ -84,7 +87,7 @@ export const registerFormSchema = loginFormSchema
 			ctx.addIssue({
 				code: 'custom',
 				path: ['confirmPassword'],
-				message: 'Passwords must match.',
+				message: 'Passwords don’t match',
```

Comment:
Should be localised string:
```suggestion
				message: $_('Passwords don’t match'),
```